pattern-library/sass/patterns/_icons.scss

```diff
@@ -6,7 +6,6 @@
     color: currentColor;
     height: auto;
     width: auto;
-    font-family: $icon-font;
```

Comment:
In the investigation I had with @schenedx, he saw the .icon has the edx-icon font in there, and it overrides the "fa" class, as a result, no FontAwesome icons show up. What @schenedx did is he added new scss files for his project using the new UXPL.

What we found is the $icon-font: 'edx-icons', 'Open Sans', sans-serif !default;
 still exists in UXPL. I forgot to remove it.